operators/src/processing/raster_vector_join/aggregator.rs

```diff
@@ -19,8 +19,10 @@ pub trait Aggregator {
 
     fn add_null(&mut self, feature_idx: usize);
 
+    #[allow(dead_code /*is used in tests*/)]
```

Comment:
would it make sense to move this to a separate trait?

Reply:
maybe. But then it is still never used, right?

Reply:
could be a trait implemented in a mod that is only compiled for tests

Reply:
i guess it is ok tu use the fields directly in test

Reply:
[18a463c](https://github.com/geo-engine/geoengine/pull/973/commits/18a463c28893769d7ab3de9e686998b2792ed907)